epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -53,69 +55,82 @@
 	<body>
 		<section id="abstract">
 			<p>The <a href="https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32019L0882">European
-					Accessibility Act (EAA)</a> is an EU directive that establishes binding accessibility targets that
+					Accessibility Act (EAA)</a> is an EU directive that establishes binding accessibility requirements that
 				must be met by many different types of products and services to strengthen the rights of people with
 				disabilities to access goods and services, including ebooks, dedicated reading software, ereading
-				devices, and ecommerce.</p>
+				devices, digital rights management softwares and ecommerce.</p>
 
-			<p>The EAA has been approved on 27 June 2019, should be implemented by EU Member States within 28 June 2022
-				and will enter in force from 28 June 2025.</p>
+			<p>The EAA has been approved on 27 June 2019 and will come into force from 28 June 2025.</p>
 
-			<p>This note aims to demonstrate that the technical requirements of the European Accessibility Act related
-				to ebooks are met by the EPUB® standard.</p>
+			<p>This note aims to demonstrate that EPUB® standard meets all the technical requirements related
+				to ebooks defined by the European Accessibility Act.</p>
```

Comment:
```suggestion
			<p>This note aims to demonstrate that EPUB® standard meets all the technical requirements related
				to ebooks, as defined by the European Accessibility Act.</p>
```

It was not clear whether ebooks are defined by the European a11y act or not :-)